.goreleaser.yml

```diff
@@ -0,0 +1,35 @@
+# Make sure to check the documentation at http://goreleaser.com
+builds:
+  - main: ./cmd/hypper
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm
+      - arm64
+source:
+  enabled: true
+  name_template: '{{ .ProjectName }}-v{{ .Version }}-source'
+archives:
+    # Default template uses underscores instead of -
+  - name_template: "{{ .ProjectName }}-v{{ .Version }}-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
+    replacements:
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
+      386: i386
+      amd64: x86_64
+checksum:
+  name_template: '{{ .ProjectName }}-v{{ .Version }}-checksums.txt'
+snapshot:
+  name_template: "{{ .Tag }}-next"
+changelog:
+  sort: asc
+  filters:
+    exclude:
```

Comment:
we might also want to exclude the third party folders...

Reply:
You mean the `internal/third_party` dir? That is still part of hypper isnt it? And if there is a bug that is fixed by changing something in the folder (updating a file, adding a new one) dont we want to have it reflected on the changelog?

Reply:
IMHO I would say no, `third_party` should only contain raw third party code without changes, everything else would be moved to `pkg` and we would reflect that in the changelog.

Reply:
sure, but for example we currently have a `urlutil.go` file there. If there is a bug in that file (upstream at helm) and we update it locally to match the updated file...should that be in the changelog in order to identify in which version was that change introduced?

Reply:
The only changes to the third party folder now would be things like us syncing changes from upstream or making an intentional change, right? Shouldn't those be in the changelog?

Reply:
I think so, because in the end its part of what we have in our repo, so its our responsability. I would not exclude those from the changelog.

Reply:
@Itxaka I guess you are right, just include them...